libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
+        properties[field["name"]] = {
+            "type": "string",
+            "index": "analyzed"
+        }
+    elif field.get("type") == "long":
+        properties[field["name"]] = {
+            "type": "long",
+            "doc_values": "true"
+        }
     elif field.get("type") == "float":
         properties[field["name"]] = {
             "type": "float",
```

Comment:
Don't we already have this here? https://github.com/ruflin/beats/blob/template-generation/libbeat/scripts/generate_template.py#L140